src/main/java/com/artipie/VertxMain.java

```diff
@@ -128,6 +129,36 @@ public static void main(final String... args) throws IOException, ParseException
         Logger.info(VertxMain.class, "Artipie was started on port %d", port);
     }
 
+    /**
+     * Find artipie settings.
+     * @param path Settings path
+     * @return Settings instance
+     * @throws IOException On read error
+     */
+    private static Settings settings(final Path path) throws IOException {
```

Comment:
@g4s8 I'd suggest to extract this method into class and properly test it: now neither `JavaResource` no this method are tested